src/vos/evtree.c

```diff
@@ -955,6 +955,26 @@ evt_desc_free(struct evt_context *tcx, struct evt_desc *desc, daos_size_t size)
 	return rc;
 }
 
+static int
+evt_node_entry_free(struct evt_context *tcx, struct evt_node_entry *ne)
+{
+	struct evt_desc	*desc;
+	int		 rc;
+
+	if (ne->ne_child == 0)
+		return 0;
+
+	desc = evt_off2desc(tcx, ne->ne_child);
+	vos_dtx_degister_record(evt_umm(tcx), desc->dc_dtx,
```

Comment:
is this always called? I think we should be able to bypass these functions for performance benchmark right?

Reply:
If the DTX is not enabled or the target is not replicated object, then vos_dtx_degister_record() will return directly. Does it will affect performance too much?

Reply:
Sorry, the vos_dtx_degister_record() will be noop when DTX is disabled.